finality-provider/cmd/fpd/daemon/daemon_commands.go

```diff
@@ -446,6 +446,71 @@ func runCommandAddFinalitySig(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// CommandEditFinalityDescription edits description of finality provider
+func CommandEditFinalityDescription() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:     "edit-finality-provider [btc_pk]",
+		Aliases: []string{"efp"},
+		Short:   "Edit finality provider",
+		Example: fmt.Sprintf(`fpd edit-finality-provider --daemon-address %s`, defaultFpdDaemonAddress),
```

Comment:
maybe it's better to provide an example including one of the editable fields, e.g., moniker